files/active_record_relation_patch.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveRecord
+  class Relation
+    def to_s
+      "Array of #{self.count} #{self.model.to_s.pluralize}"
+    end
+  end
+end
```

Comment:
Another one of these thingies. Odd, your editor should automatically take care of this — might want to check settings?

Reply:
Yeah my settings got kind of busted yesterday. I'm working on fixing it now.